python/ray/air/predictors/integrations/huggingface/huggingface_predictor.py

```diff
@@ -99,6 +99,9 @@ def _convert_data_for_pipeline(
             columns = columns[0]
         return columns
 
+    def _predict_pandas(self, data: pd.DataFrame, **kwargs) -> pd.DataFrame:
+        pass
```

Comment:
NotImplementedError should only be used for base classes right? None of these classes are base classes. Predictor is the only base class and `NotImplementedError` is being used correctly there.

Reply:
I think pass is the correct usage here.

We explicitly want to say that this abstract method should be a no-op (since `predict` is being overridden instead), rather than this abstract method is not implemented. In future PRs, we will change the predictor implementations to implement `_predict_pandas`.

Reply:
Let's remove these (make it not abstract method).